Lab1.ipynb

```diff
@@ -0,0 +1,607 @@
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "probability associated with sigma= 1.02 is 0.3461357696272652\n"
+     ]
+    }
+   ],
+   "source": [
+    "import scipy\n",
+    "from scipy import stats\n",
+    "z=1.02\n",
+    "prob= stats.norm.cdf(z)-0.5\n",
+    "print('probability associated with sigma=',z,'is', prob)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 148,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "probability associated with sigma= 0.08 is 0.03188137201398744\n"
+     ]
+    }
+   ],
+   "source": [
+    "z=0.08\n",
+    "prob= stats.norm.cdf(z)-0.5\n",
+    "print('probability associated with sigma=',z,'is', prob)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 149,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "probability associated with sigma= 3.01 is 0.4986937615512306\n"
+     ]
+    }
+   ],
+   "source": [
+    "z=3.01\n",
+    "prob= stats.norm.cdf(z)-0.5\n",
+    "print('probability associated with sigma=',z,'is', prob)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "C.To do the inverse, determine the associated 'sigma' value for a given probability, we use the percent point function (.ppf())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 150,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "For the given probability= 0.539827837277029 the associated 'sigma'= 0.10000000000000002\n"
+     ]
+    }
+   ],
+   "source": [
+    "z=0.1\n",
+    "prob_in=stats.norm.cdf(z)\n",
+    "sigma = stats.norm.ppf(prob_in)\n",
+    "print('For the given probability=',prob_in,\"the associated 'sigma'=\",sigma)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 151,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "For the given probability= 0.9793248371339299 the associated 'sigma'= 2.039999999999999\n"
+     ]
+    }
+   ],
+   "source": [
+    "z=2.04\n",
+    "prob_in=stats.norm.cdf(z)\n",
+    "sigma = stats.norm.ppf(prob_in)\n",
+    "print('For the given probability=',prob_in,\"the associated 'sigma'=\",sigma)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 152,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "For the given probability= 0.9999997133484281 the associated 'sigma'= 4.999999999970176\n"
+     ]
+    }
+   ],
+   "source": [
+    "z=5\n",
+    "prob_in=stats.norm.cdf(z)\n",
+    "sigma = stats.norm.ppf(prob_in)\n",
+    "print('For the given probability=',prob_in,\"the associated 'sigma'=\",sigma)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
```

Comment:
Mention that the sigma values you calculated from the probabilities are what you were expecting.